packages/tangle/booker_scenario_2_test.go

```diff
@@ -0,0 +1,348 @@
+package tangle
```

Comment:
The PastMarkers and FutureMarkers of message 5 in scenario 2 figure are incorrect.

The PastMarkers should be 2,3, and FutureMarkers should be 2,4.